solr/bin/solr

```diff
@@ -729,6 +733,7 @@ function jetty_port() {
 # useful for doing cross-platform work from the command-line using Java
 function run_tool() {
 
+  # shellcheck disable=SC2086
   "$JAVA" $SOLR_SSL_OPTS $AUTHC_OPTS ${SOLR_ZK_CREDS_AND_ACLS:-} -Dsolr.install.dir="$SOLR_TIP" \
```

Comment:
I tried to quote `$AUTHC_OPTS` but when that var is undefined/empty, it leaves two quotes `''` which trips the startup. So it is safest to just leave these here. Besides, these vars may intentionally contain multiple arguments that would be wrong to pass as one quoted argument.